stack.yaml

```diff
@@ -0,0 +1,6 @@
+flags: {}
+packages:
+- '.'
+extra-deps:
+- cabal-helper-0.5.1.0
```

Comment:
Why does this need to be specified here? I thought stack would just pull that from the cabal file?